dim/lib/dim/commands/export.rb

```diff
@@ -3,6 +3,7 @@
 
 require_relative '../globals'
 require_relative '../exporter/exporterInterface'
+require_relative '../exporter/raw_rst'
```

Comment:
Requirements are missing for that as well as documentation fixes, architecture, FMEA adaptions, etc...
Is function & branch coverage still 100%?

Reply:
Coverage is still 100% according to SimpleCOV.
The tests I introduced fully cover the code.

I can add the requirements, no problem I just need some guidance on where to put them, and wether or not all of the levels (i.e. SRS, SWA and SMD are needed, or only SRS).

I'll update the architecture diagram and add the documentation for the RAW RST exporter. Do you think the FMEA needs to be adapted? Since this exporter doesn't add anything new but just follows the same pattern that the other exporter have....

Reply:
This is now done. I added the requirements, updated the architectural diagram and documented the raw-rst format in `cli.rst`. I also linked the tests with the added requirements.

Branch coverage was under 100% but that is now fixed, I added an extra test case to cover the missing alternative.